webant/templates/recents.html

```diff
@@ -0,0 +1,90 @@
+{% extends "bootstrap/base.html" %}
+{% import "bootstrap/fixes.html" as fixes %}
+
+{% block title %}
+Libreant | {%trans%}Recents{%endtrans%}
+{% endblock %}
+
+{% block styles %}
+ {{super()}}
+<link href="{{ url_for('static', filename='item_list.css') }}" rel="stylesheet">
+<style>
+header{
+    margin-top:10px;
+    margin-bottom: 20px;
+}
+header h1{
+    margin-top:0px;
+    margin-bottom: 0px;
+    font-size: 2.3em;
+    
+}
+</style>
+{% endblock %}
+
+{% block navbar %}
+{% import 'navbar.html' as navbar %}
+{{ navbar.navbar(search=True) }}
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    <header>
+        <h1>{%trans%}Recently added items{%endtrans%}</h1>
+    </header>
+
+    {% if not items %}
+
+    {%trans%}Sorry, no items added since now{%endtrans%}
+
+    {% else %}
+
+    <div id="item-list">
+        {% for b in items %}
+        <div class="item-div row" onclick="window.open('{{url_for('view_book', bookid=b['_id'])}}')">
```

Comment:
> onlick is deprecated. The best way to do this is putting some js inside a proper script
> 
> I did not find anything online. Can you provide some resources

I already did it on the above comment, but I'll be clearer
1. create a new js file in the `static` directory. Since we're at it, we can use this file for everything we need, so let's give it a generic name, like `ui-main.js`
2. write this thing inside it
```js
jQuery(function($) {
$('.item-div').click(function(evt) {
    document.location.href = $('a.item-title', evt.target).prop('href');
})
```
3. make sure to include the script
4. remove the `onclick` attribute on `item-div`